### DISCUSSION TITLE
Getting AutoCompletion Working for FontAwesomeIcon Component

### DISCUSSION THREAD

**Discussion by st11x** [vuejs/language-tools#1713] [Q&A]
Hi,

I am using vue-awesome with vue3. Instead of declaring the component globally, I had imported it locally in my SFC like this

`import { FontAwesomeIcon }  from '@fortawesome/vue-fontawesome'`

However, it doesn't suggest this as a component name, nor are the props offered as suggestions. 

![image](https://user-images.githubusercontent.com/7398499/184190080-2a95f842-1efe-4802-9666-fe6cec0247b0.png)

and it also doesn't suggest "icon" as a prop.

![image](https://user-images.githubusercontent.com/7398499/184190278-59919c93-c000-4b1a-bade-88ff30896db4.png)

Are you getting the same? However, the code does work if you fill it out completely yourself. 

The autocompletion works (updated: mostly) for other libraries like HeadlessUI, so I am not sure what am I missing here.

thanks
  Matt

**Comment by johnsoncodehk** [comment]
Seems `@fortawesome/vue-fontawesome` define `FontAwesonIcon` as empty: https://github.com/FortAwesome/vue-fontawesome/blob/db8a567ae02c8502ded2bfeb1f9646ce52553862/index.d.ts#L3

Maybe also have to wait for https://github.com/FortAwesome/vue-fontawesome/issues/400 fix.
